doc/sphinx/src/getting-started.rst

```diff
@@ -11,12 +11,13 @@ At it's most basic, you can download and compile ``singularity-eos`` with:
   cd singularity-eos
   mkdir bin
   cd bin
-  cmake ..
+  cmake -DSINGULARITY_FORCE_SUBMODULEMODE=ON -DSINGULARITY_USE_FORTRAN=OFF ..
```

Comment:
I think we should also direct to the location where more detailed configure/build information is supplied.